### DISCUSSION TITLE
Remake "Contribute to Sponsorblock" icon in video play to match the style of the other icons

### DISCUSSION THREAD

**Discussion by Fxsch** [libre-tube/LibreTube#4934] [General]
### Describe your suggested feature

Here's how it currently looks:
![Screenshot_20231008-153159_LibreTube](https://github.com/libre-tube/LibreTube/assets/77849217/47ed5e81-65e4-44ab-a146-2e15680923ec)

I could remake the icon myself but I don't know how to make LibreTube actually use it. If someone could tell me how, I'd do it.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by IndusAryan** [comment]
u can paste ur icon image here , let the developer will decide if its replacable, if the maintainer says ok, i will convert it in drawable or find some other icon similar to that and change it

**Comment by Bnyro** [comment]
Converting to discussion since it's awaiting a svg/design suggestion.

**Comment by Fxsch** [comment]
I found where the icons are saved (important as I need a reference). That means I could also just make a pull request myself. I'll still upload the icons here though, as I'm planning to make slight changes that might need some discussion.

Here's the first one (Upload segment icon):
![ic_upload_segment](https://github.com/libre-tube/LibreTube/assets/77849217/624897ad-d08b-4c31-b17a-177d6a1cf2b3)

Changes:
- It has no inner fill anymore, as other icons only have outlines as well
- I changed the arrow to the one that is used in the Material Design 3 icons as well, as that fits into the style, while the old one doesn't
- All lines have the same width as the lines in the other icons

**Reply by Fxsch** [reply]
Just noticed I forgot to make the lines white but I can just change that when I actually open the pull request

**Comment by Fxsch** [comment]
Here's the second one (SponsorBlock enabled):
![ic_sb_enabled](https://github.com/libre-tube/LibreTube/assets/77849217/d32338f8-1531-490c-bcab-956c1ff25112)

Changes:
- Also changed to only outline and the same line width
- Instead of the strike-through circle in the middle, there's a line going through the entire icon, as that's what the other Material Design 3 icons do as well

**Comment by Fxsch** [comment]
Here's the last one (SponsorBlock disabled):
![ic_sb_disabled](https://github.com/libre-tube/LibreTube/assets/77849217/37a1d9fa-cf10-48de-8643-6d44d902855e)

Changes:
- Outline and line width as was the case with the other 2
- Instead of a heart, which really didn't convey well what the button does, there's a check mark, which, as you might have guessed, is the one the other Material Design 3 icons use as well

**Comment by Fxsch** [comment]
If there's no issue with these icons, tell me and I'll open a pull request right away

**Comment by Bnyro** [comment]
Hey @Fxsch,

what a great job that you've done here!
I really like the design proposals and would be happy to merge your pull request once it's there.

Cheers

Bnyro

**Comment by Fxsch** [comment]
I opened the pull request (#5014)
